sdk-python/livescoreapi.py

```diff
@@ -7,17 +7,62 @@
 """
 
 import urllib.request
-
+import json
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
     
-    def get_all(key,secret):
-        html_file = urllib.request.urlopen("http://livescore-api.com/api-client/scores/live.json?key=" + key + "&secret=" + secret)
-        json_data = html_file.read()
-        return json_data
+    Methods
+    -------
+    _get_html_file(link)
     
-
+    get_all()
+    
+    """
+    link_get_all = "http://livescore-api.com/api-client/scores/live.json?key={}&secret={}"
+   
+    def __init__(self, key, secret):
         
-                
+        try: 
+            key.strip()
+            secret.strip()
+            assert len(key) == 16
+            assert len(secret) == 32
+            self.key = key
+            self.secret = secret 
+            
+        except Exception as e:
+            print("Key or Secret is Empty!")
```

Comment:
No Need to catch the exception. The idea of it is to allow the user to handle it the way they wish